### PR TITLE
feat: make routes localized

### PR DESCRIPTION
**Pull request by fparga**
## Description

Add language to URL. i.e `/#/chat` -> `/#/en/chat/`

## Related issues

Closes dialoguemd/covidflow#68

## Checklist

- [X] PR title follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) guidelines 
- [X] All relevant PR sections are populated, irrelevant ones are removed

### CODE REVIEW

File: src/routes/rasa.tsx [hunk 1]
```diff
@@ -43,22 +32,26 @@ function renderDefault(props: {}): JSX.Element {
...
-      <Route path={`${path}/ci`} render={renderCheckin} />
-      <Route path={path} render={renderDefault} />
+      <Route exact path={path}>
+        <Rasa />
+      </Route>
+      <Route exact path={`${path}/ci/:rid`}>
+        <RasaCheckin />
+      </Route>
+      <Redirect to={path} />
     </Switch>
```
Comment (mathieubergeron) [inline src/routes/rasa.tsx:55]: :+1: those changes are in lines with what we've talked about. thanks!

**Comment by vicrep** [comment]
FYI @fparga this was already achievable, and is natively behaviour of our i18n library: 

https://covid19.dialogue.co/?lng=fr#/

https://covid19.dialogue.co/?lng=en#/

**Comment by fparga** [comment]
@vicrep I don't mind reusing this mechanism if you think it's the way to go, but right now there are some issues with it.

- there is no persistence of the chosen language if you refresh the page
- if you pass the `lng` query param, it's always there even if you then switch the language

**Comment by vicrep** [comment]
I just would rather use default / already implemented behaviour as opposed to doing our own version of it, one less thing to maintain.

There actually is persistence built in (using cookies and/or localStorage), but I believe default precedence is given to queryString. This can trivially be altered through configuration of the browser detection plugin we use: https://github.com/i18next/i18next-browser-languageDetector#detector-options 

This is where it's configured in this app: https://github.com/dialoguemd/covid-19/blob/master/src/services/i18n.ts#L31

It also looks like browser detection caching / persistence was disabled for some reason, feel free to re-nable it, I think that was just a bad copy-pasta from another project.

**Comment by fparga** [comment]
Sounds good, I'll try to configure the plugin in a way that makes more sense then. 
Closing this, I'll open a new PR since it won't have much code in common.

**Comment by vicrep** [comment]
Thank you! Let me know if you run into issues. If it doesn't work, then feel free to re-open this PR :)

**Pull request closed without merging**